### ISSUE TITLE
Problems with NVDA and adding people to a Skype conference

### ISSUE THREAD

**Issue by CristinaAccardi** [nvaccess/nvda#11580]
Good morning. My name is Cristina and I'm from Italy. I'm writing because I'm having some problems with NVDA and the last Skype update when it comes to adding people to a conference. First of all, I would like to say I have the last version of Windows 10, NVDA and Skype. With the last Skype update, I'm having problems to add people to a conference. After the first call, when I go to the add people dialogue, NVDA doesn't read the checkboxes with the names of my contacts, like it did before. I contacted Microsoft, because I thought it was a Skype problem. I talked to the disability assistance department and they told me to try both with NVDA and with Narrator. I couldn't do anything successfully untill yesterday I talked to a senior engineer and he activated the Narrator scann mode for me. It was actually the only way I could read those checkboxes. May I ask you to have a combined phone session with this engineer, in order to understand why this happens? I thank you from now for your attention and I'm looking forward to read your answer. Best regards.
Cristina Accardi

#### Did you try to run the COM registry fixing too

**Comment by Adriani90** [comment]
cc: @michaelDCurran

**Comment by CristinaAccardi** [comment]
Why is nobody answering?

**Comment by Adriani90** [comment]
This is fixed as per skype 94 and NVDA 2023.1 RC1. The checkboxes are being reported correctly, as well as the names of contacts in the list. Closing as works for me.
